### DISCUSSION TITLE
Porting project to platform io

### DISCUSSION THREAD

**Discussion by rbagdazian** [pschatzmann/arduino-audio-tools#670] [Q&A] [answered]
I have successfully built and run the basic-a2dp-audiokit.ino sketch with the Arduino framework and was able to hear music from my mobile phone playing over headphones. I then attempted to build the same sketch with platformio in the VSCode editor and after adding in all the library dependencies, I got the following diagnostics:
Dependency Graph
|-- audio-tools @ 0.9.3+sha.4fade9e
|   |-- WiFiClientSecure @ 2.0.0
|   |   |-- WiFi @ 2.0.0
|   |-- WiFi @ 2.0.0
|   |-- ESP32-A2DP @ 1.7.1+sha.5f87ad2
|   |-- audiokit @ 0.6.2+sha.eed74a4
|   |   |-- SPI @ 2.0.0
|   |-- SPI @ 2.0.0
|-- ESP32-A2DP @ 1.7.1+sha.5f87ad2
|-- audiokit @ 0.6.2+sha.eed74a4
|   |-- SPI @ 2.0.0
|-- FastLED @ 3.5.0+sha.3d2ab78
|   |-- SPI @ 2.0.0
Building in release mode
Compiling .pio\build\esp32cam\lib9b1\audiokit\audio_hal\i2c_wire.cpp.o
.pio/libdeps/esp32cam/audiokit/src/audio_hal/i2c_wire.cpp:13:10: fatal error: Wire.h: No such file or directory

**************************************************************
* Looking for Wire.h dependency? Check our library registry!
*
* CLI  > platformio lib search "header:Wire.h"
* Web  > https://registry.platformio.org/search?q=header:Wire.h
*
**************************************************************

 #include <Wire.h>
          ^~~~~~~~
compilation terminated.

When I configured the project I selected the arduino framework. The search of the library registry didn't seem to be much help.

Any thoughts on how to solve this problem? Much Appreciated!

-Richard

**Comment by pschatzmann** [comment, marked as answer]
I think the error message is pretty clear: you did not add all necessary libraries: You forgot to add the Wire library!
https://community.platformio.org/t/wire-h-not-found-platformio-latest-version/2046

**Reply by rbagdazian** [reply]
Thanks very much!
